airflow/ui/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -18,7 +18,8 @@
  */
 import { Box, Heading } from "@chakra-ui/react";
 
-import { Health } from "./Health";
+import { Health } from "./Health/Health";
+import { Stats } from "./Stats/Stats";
```

Comment:
Let's add a `/Stats/index.ts` file so the import statements are a little cleaner